documentation/source/user_section/tutorials/non-template-registration/contrast-agnostic-registration/preprocessing-t1.rst

```diff
@@ -0,0 +1,76 @@
+.. _preprocessing-t1:
+
+Preprocessing steps to highlight the spinal cord (T1w)
+######################################################
+
+Next, we repeat the exact same preprocessing steps that were completed for the T2w image.
+
+Switching to the T1 directory
+-----------------------------
+
+Since we are now working with a different contrast, we will need to change directories.
+
+.. code::
+
+   cd ../t1
+
+Generating a spinal cord segmentation
+-------------------------------------
+
+Next, we obtain a segmentation image that highlights the spinal cord.
+
+.. code::
+
+   sct_deepseg_sc -i t1.nii.gz -c t1 -qc ~/qc_singleSubj
+
+:Input arguments:
+   - ``-i`` : The input image.
+   - ``-c`` : The contrast of the input image.
+   - ``-qc`` : Directory for Quality Control reporting. QC reports allow us to evaluate the results slice-by-slice.
+
+:Output files/folders:
+   - ``t1_seg.nii.gz`` : A mask image containing the segmented spinal cord.
+
+Creating a mask around the spinal cord
+--------------------------------------
+
+Once the segmentation is obtained, we can use it to create a mask around the cord.
+
+.. code::
+
+   sct_create_mask -i t1.nii.gz -p centerline,t1_seg.nii.gz -size 35mm -f cylinder -o mask_t1.nii.gz
```

Comment:
This is out of scope for this PR, but I'll note that `sct_create_mask` generates a couple of deprecation warnings, coming from how we use `nibabel`:
```
.../spinalcordtoolbox/spinalcordtoolbox/scripts/sct_create_mask.py:227: DeprecationWarning: get_header method is deprecated.
Please use the ``img.header`` property instead.

* deprecated from version: 2.1
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 4.0
  hdr = centerline.get_header()  # get header
/home/mguaypaq/neuropoly/spinalcordtoolbox/spinalcordtoolbox/scripts/sct_create_mask.py:230: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data_centerline = centerline.get_data()  # get centerline
```

Reply:
Gosh... --> https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3736#issuecomment-1070991573

I really, really want to start cleaning up the deprecation warnings? It's a hard thing to triage/prioritize, because maintenance tasks often fall by the wayside, but it's been such a big itch every time I run the test suite. Selfishly, I very much want to make that warning count go down. It feels very fun/satisfying to me, in much the same way fixing the linting issues were (https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3842).